test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.server.cs

```diff
@@ -112,30 +115,28 @@ public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes(bool? enableGrpcA
     [Theory(Skip = "Skipping for .NET 6 and higher due to bug #3023")]
 #endif
     [InlineData(null)]
-    [InlineData(true)]
-    [InlineData(false)]
-    public void GrpcAspNetCoreInstrumentationAddsCorrectAttributesWhenItCreatesNewActivity(bool? enableGrpcAspNetCoreSupport)
+    [InlineData("true")]
+    [InlineData("false")]
+    [InlineData("True")]
+    [InlineData("False")]
+    public void GrpcAspNetCoreInstrumentationAddsCorrectAttributesWhenItCreatesNewActivity(string enableGrpcAspNetCoreSupport)
```

Comment:
Not related to this PR but these tests should be moved to `OpenTelemetry.Instrumentation.AspNetCore.Tests`.